### DISCUSSION TITLE
[Feature] Api granular access control per Album

### DISCUSSION THREAD

**Discussion by bestrocker221** [immich-app/immich#13347] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

First of all thanks for the amazing project.
I was wondering if support for API key granular access control is in the roadmap.

I am asking because it would be great to have control over what albums/photos can be accessed via api key.

Assume a Immich frame/kiosk or a custom implemented api consumer, I would want to limit the access to such an api key, and not give out the whole library access to it.

ex: API key 1: read only access to album X

Thanks again

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#6910
